web/templates/attendance/fill_course_attendance.html.eex

```diff
@@ -71,6 +71,56 @@
                               <i class="material-icons attendance-form__attendance attendance--absent">add_circle</i>
                               <i class="material-icons attendance-form__attendance attendance--not-filled">radio_button_unchecked</i>
                             </button>
+                            <button
+                              type="button"
+                              data-attendance-id="<%= fh.id %>"
+                              class="mdl-button mdl-js-button mdl-button--icon show-attendance-modal"
+                            >
+                              <i
+                                class="
+                                  material-icons
+                                  attendance-comment__icon
+                                  attendance-comment__icon--fill
+                                  <%=
+                                    if fh.data.comment && String.length(fh.data.comment) > 0 do
```

Comment:
you can extract this condition to a function in view and then use "Changeset.get_field(changeset, key, default \\ nil)" instead of "fh.data"